api/AmbientLightSensor.json

```diff
@@ -34,36 +84,74 @@
           "ie": {
             "version_added": false
           },
-          "opera": {
-            "version_added": "56"
-          },
-          "opera_android": {
-            "version_added": "48"
-          },
-          "safari": {
-            "version_added": false
-          },
-          "safari_ios": {
-            "version_added": false
-          },
-          "samsunginternet_android": [
+          "opera": [
             {
-              "version_added": "6.0"
+              "version_added": "49",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor",
+                  "value_to_set": "Enabled"
+                },
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor-extra-classes",
+                  "value_to_set": "Enabled"
+                }
+              ]
             },
             {
-              "version_added": "10.0",
-              "notes": "Based on Generic Sensor API."
+              "version_added": "43",
+              "version_removed": "49",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor",
+                  "value_to_set": "Enabled"
+                }
+              ]
             }
           ],
-          "webview_android": [
+          "opera_android": [
             {
-              "version_added": "54"
+              "version_added": "46",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor",
+                  "value_to_set": "Enabled"
+                },
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor-extra-classes",
+                  "value_to_set": "Enabled"
+                }
+              ]
             },
             {
-              "version_added": "69",
-              "notes": "Based on Generic Sensor API."
+              "version_added": "43",
+              "version_removed": "46",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor",
+                  "value_to_set": "Enabled"
+                }
+              ]
             }
-          ]
+          ],
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
```

Comment:
I tested https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/AmbientLightSensor in Samsung Internet to confirm. This seems like the kind of API that some vendor might have chosen to experiment with and ship earlier than Chrome, but not this time.